### PR TITLE
[docs] Fix the broken training doc link for Textual Inversion

### LINKED ISSUE #5387: Broken Link in Textual Inversion Documentation
Issue Description:
The link to the [Textual Inversion Training Guide](https://[huggingface.co/docs/diffusers/using-diffusers/training/text_inversion](https://huggingface.co/docs/diffusers/using-diffusers/training/text_inversion)) is currently broken on [this](https://huggingface.co/docs/diffusers/using-diffusers/textual_inversion_inference) page. 

Broken Link:
https://[huggingface.co/docs/diffusers/using-diffusers/training/text_inversion](https://huggingface.co/docs/diffusers/using-diffusers/training/text_inversion)

I am willing to submit a Pull Request (PR) to correct this link if it that's fine.

### PR DESCRIPTION
**Pull request by mhetrerajat**
# What does this PR do?

Fixes #5387 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/en/using-diffusers/textual_inversion_inference.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 The [`StableDiffusionPipeline`] supports textual inversion, a technique that enables a model like Stable Diffusion to learn a new concept from just a few sample images. This gives you more control over the generated images and allows you to tailor the model towards specific concepts. You can get started quickly with a collection of community created concepts in the [Stable Diffusion Conceptualizer](https://huggingface.co/spaces/sd-concepts-library/stable-diffusion-conceptualizer).
 
-This guide will show you how to run inference with textual inversion using a pre-learned concept from the Stable Diffusion Conceptualizer. If you're interested in teaching a model new concepts with textual inversion, take a look at the [Textual Inversion](./training/text_inversion) training guide.
+This guide will show you how to run inference with textual inversion using a pre-learned concept from the Stable Diffusion Conceptualizer. If you're interested in teaching a model new concepts with textual inversion, take a look at the [Textual Inversion](../training/text_inversion.md) training guide.
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/textual_inversion_inference.md:7]:
```suggestion
This guide will show you how to run inference with textual inversion using a pre-learned concept from the Stable Diffusion Conceptualizer. If you're interested in teaching a model new concepts with textual inversion, take a look at the [Textual Inversion](../training/text_inversion) training guide.
```

**Review by stevhliu — COMMENTED** [review]
Thanks for the fix! To get the CI tests to pass, can you run `make fix-copies` and also update your branch with the latest updates on `main`? 🙂

File: docs/source/en/using-diffusers/textual_inversion_inference.md (hunk 1 above)
Comment (mhetrerajat) [reply docs/source/en/using-diffusers/textual_inversion_inference.md]: addressed in ed82bf94aab0351ba24bff135d377b2778aba316

File: src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py [hunk 2]
```diff
@@ -106,7 +106,6 @@ class AltDiffusionPipeline(DiffusionPipeline, TextualInversionLoaderMixin, LoraL
         feature_extractor ([`~transformers.CLIPImageProcessor`]):
             A `CLIPImageProcessor` to extract features from generated images; used as inputs to the `safety_checker`.
     """
-
```
Comment (stevhliu) [inline src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py:109]: Lets keep the blank line here, it is causing the test to fail (same for the `pipeline_alt_diffusion_img2img.py` file).

Comment (mhetrerajat) [reply src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py:109]: I believe the change is automatically generated by the "make fix-copies" command. I will investigate why it's being added and remove the commit that made this change.

Comment (stevhliu) [reply src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py:109]: Ah yeah, I had a similar issue and I resolved it by removing the blank lines and not running the `make fix-copies` command.

**Pull request closed without merging**